dds/DCPS/StaticDiscovery.cpp

```diff
@@ -87,7 +87,6 @@ StaticEndpointManager::StaticEndpointManager(const RepoId& participant_id,
 #endif
   , max_type_lookup_service_reply_period_(0)
   , type_lookup_service_sequence_number_(0)
-  , use_xtypes_(true)
   , use_xtypes_complete_(false)
```

Comment:
```suggestion
```

Reply:
Unlike `use_xtypes_`, this is tied to the rest of the leftover XTypes stuff in `StaticDiscovery`, should I pull it all out?

Reply:
Yes, unless that proves to be difficult

Reply:
Yeah, all that stuff was from me not being sure whether we were going to try and do anything with static discovery and XTypes. We decided we weren't so pull it out if possible.  There's some weirdness with functions needing to be in both discoveries so it might not be the easiest thing to do.